style.css

```diff
@@ -14,6 +14,10 @@
   justify-content: space-between;
 }
 
```

Comment:
- [ ] As suggested by the previous reviewer,  I kindly suggest that you make use of the font used in your template design which is [poppins](https://fonts.google.com/specimen/Poppins). 

You can import Poppins into your CSS file by writing this at the top of your CSS file:
`@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap');
`